build_tools/conda_qt5_win_commercial.yml

```diff
@@ -25,7 +25,7 @@ dependencies:
  - pyzmq=19.0.1
  - mkl=2020.0
  - pip:
-   - matplotlib==2.2.3
+   - matplotlib==3
```

Comment:
Not a specific version? The current release is 3.4.3 from 2021-08-13 on PyPI

Reply:
precisely